app/src/app/components/about-page/about-page.component.scss

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.about {
+  &-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 40px;
+  }
+
+  &-header {
+    &-container {
+      display: flex;
+      flex-direction: row;
+      align-items: flex-end;
+      justify-content: space-between;
+      padding: 32px;
+    }
+
+    &-divider {
+      width: 100%;
+      margin-top: 8px;
+      margin-bottom: 32px;
+    }
+
+    &-text {
+      width: 100%;
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+
+      & h2 {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  &-label {
+    margin-top: 1rem;
+
+    & h4 {
+      margin: none;
+    }
+  }
+
+  &-description {
+    margin-top: 32px;
+    padding-right: 32px;
+    padding-left: 32px;
+  }
+
+  &-operation {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    & h3 {
+      margin: 0;
+    }
+
+    & iframe {
+      width: 100%;
+      height: 640px;
+      border: none;
+      margin-right: auto;
+      margin-left: auto;
+      margin-top: 16px;
+    }
+  }
+
+  &-team {
+    &-container {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      justify-content: flex-start;
+    }
+
+    &-member {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      padding: 16px;
+
+      & a,
+      p {
+        text-align: center;
+      }
+    }
+
+    &-picture-container {
+      height: 164px;
+      width: 164px;
+      padding: 8px;
+      margin-bottom: 8px;
+
+      & img {
+        height: 100%;
+        width: 100%;
+        border-radius: 100%;
+      }
+
+      &:not(:last-child) {
+        margin-right: 16px;
+      }
+    }
+  }
+}
+
+@media (max-width: 768px) {
```

Comment:
Might be a good idea to add this to a scss variables file so it can be used by other component styles if needed and doesn't get duplicated.

Reply:
Will do, I've been using the same breakpoint throughout so it should clean up things

Reply:
After looking into this, I don't think there's a simple way of using these breakpoints as variables but the standard seems to be to just have them on every styling sheet where they're used.